teuthology/test/test_packaging.py

```diff
@@ -352,7 +352,7 @@ def test_get_package_version_not_found(self, m_get, m_get_config_value,
 
     GITBUILDER_DISTRO_MATRIX = [
         ('rhel', '7.0', None, 'centos7'),
-        ('centos', '6.5', None, 'centos6_5'),
+        ('centos', '6.5', None, 'centos6'),
```

Comment:
yeah, it should be.  I'll add both centos 7.0 and centos 7.1.

This is used to feed my test that uses pytest.parameterize below.  This makes sure I can generate the correct distro name given distro, version and codename.